tasks/system_probe.py

```diff
@@ -343,6 +350,35 @@ def ninja_network_ebpf_programs(nw: NinjaWriter, build_dir, co_re_build_dir):
         ninja_network_ebpf_co_re_program(nw, infile, outfile, network_co_re_flags)
 
 
+def ninja_kernel_bug_binaries(nw: NinjaWriter, arch: str | Arch):
+    arch = Arch.from_str(arch)
+
+    # do not build for arm64
+    if arch.ci_arch == "arm64":
```

Comment:
```suggestion
    if arch == ARCH_ARM64
```

(import from `tasks.libs.types.arch`)